features/step_definitions/map_steps.rb

```diff
@@ -104,9 +104,9 @@ def find_map_icon klass, org_id
   org_id = klass.find_by(name: name).id
   marker_class = (icon_size == "small") ? "measle" : "marker"
   if marker_class == "measle"
-    expect(find_map_icon(marker_class, org_id)["src"]).to end_with "/assets/measle.png"
+    expect(find_map_icon(marker_class, org_id)["src"]).to match /\/assets\/measle-(\w*)\.png/ix
   else
-    expect(find_map_icon(marker_class, org_id)["src"]).to end_with "/assets/marker.png"
+    expect(find_map_icon(marker_class, org_id)["src"]).to match /\/assets\/marker-(\w*)\.png/ix
```

Comment:
looking good - to replicate exactly should we include the `$` marker in the regex to indicate we expect this text to appear at the end of the src attribute as we did in the original test ...?

Reply:
@tansaku yes! I think you are right :+1: I will make the corresponding changes.